repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -407,6 +407,15 @@ def filter_out_out_pkgs(event_in_pkgs, event_out_pkgs):
     return {k: v for k, v in event_in_pkgs.items() if k not in event_out_pkgs}
 
 
+SKIPPED_PKGS_MSG = (
+    'packages will be skipped because they are available only in '
```

Comment:
typo? `The following packages` ?

Reply:
No, the whole message is constructed here: https://github.com/oamg/leapp-repository/pull/543/files#diff-a7492401bb654d7516a25e720ece7239R498
It will start with the number of pkgs, like "3 packages will be skipped ...".

Reply:
aaah. Now I see it. Thanks!